makelib/golang.mk

```diff
@@ -18,8 +18,8 @@
 # Optional. The Go Binary to use
 GO ?= go
 
-# Optional. Required Go version.
-GO_REQUIRED_VERSION ?= 1.18
```

Comment:
`GO_REQUIRED_VERSION` is not referenced in any repo:
- https://github.com/search?q=org%3Aupbound+GO_REQUIRED_VERSION&type=code
- https://github.com/search?q=org%3Acrossplane+GO_REQUIRED_VERSION&type=code